index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">

```

Comment:
```suggestion
<html lang="ru" dir="ltr">
```

Сайт на русском, а тег настроен на английский.